br/pkg/streamhelper/basic_lib_for_test.go

```diff
@@ -93,13 +94,13 @@ func overlaps(a, b kv.KeyRange) bool {
 
 func (r *region) splitAt(newID uint64, k string) *region {
 	newRegion := &region{
-		rng:        kv.KeyRange{StartKey: []byte(k), EndKey: r.rng.EndKey},
-		leader:     r.leader,
-		epoch:      r.epoch + 1,
-		id:         newID,
-		checkpoint: r.checkpoint,
-		fsim:       r.fsim.fork(),
+		rng:    kv.KeyRange{StartKey: []byte(k), EndKey: r.rng.EndKey},
+		leader: r.leader,
+		epoch:  r.epoch + 1,
+		id:     newID,
+		fsim:   r.fsim.fork(),
 	}
+	r.checkpoint.Store(r.checkpoint.Load())
```

Comment:
Should not it be `newregion.checkpoint.Store(r.checkpoint.Load())`?

Reply:
Yes, fixed.